setup.py

```diff
@@ -27,7 +27,6 @@
 
 test_requirements = [
     'pytest==6.1.1',
-    'pytest-pylint==0.17.0',
```

Comment:
Can we re-add this as a requirement for testing? We sometimes want to be able to run linting locally without having to install the required files.

Reply:
Fixed